src/mainPage/header.ts

```diff
@@ -0,0 +1,75 @@
+import { icons, authn, initHeader } from 'solid-ui'
+/**
+ * menu icons
+*/
+const HELP_MENU_ICON = icons.iconBase + 'noun_help.svg'
+const SOLID_ICON_URL = 'https://solidproject.org/assets/img/solid-emblem.svg'
+
+/**
+ * menu elements
+*/
+const USER_GUIDE_MENU_ITEM = 'User guide'
```

Comment:
suggestion: because they are all menu elements, maybe we could have an enum.

Reply:
I think enum will not fit with having property files as a next step.